rmc-docs/src/dev-documentation.md

```diff
@@ -12,6 +12,10 @@
 ```
 ([`--keep-stage` comes with caveats](https://rustc-dev-guide.rust-lang.org/building/suggested.html#incremental-builds-with---keep-stage). Know that it may cause spurious build failures.)
 ```bash
+# Build rmc crate
+./scripts/setup/rmc_build_lib.sh
```

Comment:
Can we make this part of x.py in the rmc case?

Reply:
I debated about this one. I wanted to start untangling ourselves from the bootstrap script. My goal is to actually enhance this script to also compile an RMC friendly std library.

Reply:
I'll merge as is since this is documenting what's been pushed already. If you prefer I can modify the bootstrap script to build the RMC library and I can update this doc again. Just let me know and I can create a follow up PR.